test/unit/server/file-drop.js

```diff
@@ -1,65 +1,59 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-
-const rimraf = require('rimraf');
+const fs = require('node:fs/promises');
+const path = require('node:path');
 
 const request = require('../../setup/setup').getFileDropAgent();
 
 const homedir = path.join(__dirname, '../../homeDirectories/main');
 
 describe('drop file server', () => {
-  afterEach(() => {
-    rimraf.sync(path.join(homedir, 'test/kind1'));
-  });
-  test('api endpoint using query strings', () => {
+  afterEach(() =>
+    fs.rmdir(path.join(homedir, 'test/kind1'), {
```

Comment:
The `recursive` option on `rmdir` is deprecated since Node.js 14: https://nodejs.org/docs/latest/api/fs.html#fspromisesrmdirpath-options. You should use `rm`.